docs/requirements/elicitation/storyBoard.md

```diff
@@ -0,0 +1,42 @@
+# _StoryBoards_
```

Comment:
Tirar o itálico para padronizar com os títulos das outras técnicas.

Reply:
Feito!